### ISSUE TITLE
Set up instance of VecNet Openmalaria implementation

### ISSUE THREAD

**Issue by ThomasASmith** [SwissTPH/openmalaria#174]
An instance of the VecNet OpenMalaria implementation is needed that does not have dependencies on continued software/hardware support from Notre Dame.

**Comment by tph-thuering** [comment]
I was finally able to test https://github.com/vecnet/om.git with docker successfully.
Now on to setting this up on a production system.

**Comment by rossaman** [comment]
Brilliant that it's tested with docker. For the production system... what would be needed? Is this likely to be possible for the course this year - ie ready by mid-March ?

**Comment by sgtbigman** [comment]
Having contributed significantly to the om repository, I wanted to see how this setup was coming along. Please let me know if there is anything I can do to help.

Thanks.

**Comment by tph-thuering** [comment]
@rossaman Sorry for taking so long. The issue is that I could not catch up with the responsible people at our Organisation for setting up a legit server. Maybe some things have changed by now? Is this still something you need for the continuation of the courses? Could you send me an email if you know anyone who can setup Linux-servers which is allowed under our policy?

**Comment by ThomasASmith** [comment]
We still need this.  So far as I understand it the Swiss TPH ticket is still sitting there acquiring dust.
I am not sure what Organisation @tph-thuering is referring to?  We need an instance that we have some control of:  SciCore/ Unibas may be a more reliable way of achieving this than SwissTPH. What do you think @rossaman ?

**Comment by rossaman** [comment]
Funnily enough I emailed IT earlier today, I don't think it has progressed. In April, @Baguage updated the installation instructions and we submitted to IT as a project. If it would be possible on sciCORE that could be great, but I'm not sure that the students would have access to sciCORE without getting user accounts. 
The new installation instructions are on - https://github.com/vecnet/om/blob/master/INSTALLATION_INSTRUCTIONS.md
